BUILD

```diff
@@ -9,17 +9,6 @@ files(name="files", sources=["BUILD_ROOT", "pants.toml"])
 
 python_test_utils(name="test_utils")
 
-_local_environment(
-    name="default_env",
-)
-
-_local_environment(
-    name="macos_local_env",
-    compatible_platforms=["macos_arm64", "macos_x86_64"],
-    # Avoid system Python interpreters, which tend to be broken on macOS.
-    python_bootstrap_search_path=["<PYENV>"],
-)
```

Comment:
This was unused

Reply:
I use this via a `.pants.rc`, as suggested in https://github.com/pantsbuild/pants/pull/16683/files#r958723646

Reply:
You can instead set `[python-bootstrap].search_path` in `pants.rc`. There isn't much benefit to using one approach vs the other, given that the # of overridden fields here == 1.

Reply:
Got it. Thanks!